PWGDQ/TableProducer/tableMaker.cxx

```diff
@@ -96,6 +96,8 @@ struct TableMaker {
   Configurable<std::string> fConfigMuonCuts{"cfgMuonCuts", "muonQualityCuts", "Comma separated list of muon cuts"};
   Configurable<float> fConfigBarrelTrackPtLow{"cfgBarrelLowPt", 1.0f, "Low pt cut for tracks in the barrel"};
   Configurable<float> fConfigMuonPtLow{"cfgMuonLowPt", 1.0f, "Low pt cut for muons"};
+  Configurable<float> fConfigMinTpcSignal{"cfgMinTpcSignal", 1.0f, "Minimum TPC signal"};
+  Configurable<float> fConfigMaxTpcSignal{"cfgMaxTpcSignal", 1.0f, "Maximum TPC signal"};
```

Comment:
Please add meaningful defaults. Since this will be used to filter also hadrons, not just jpsi, we can use something like 30 for the minimum and 300 for maximum.